packages/charting/src/components/Legends/Legends.base.tsx

```diff
@@ -66,6 +67,7 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
         hoverAction: legend.hoverAction!,
         onMouseOutAction: legend.onMouseOutAction!,
         color: legend.color,
+        shape: legend.shape,
```

Comment:
this is where I think we can pass in some of that ratio data to get the right values and allow it to be modified a bit.

Reply:
I don't quite understand it. Could you share more details about it?

Reply:
I think the comment moved but basically pass the ratio in as a styles prop. So line 44 add a prop to the object passed in that has the ratio. Then we can do the styles calcs within the styles function

Reply:
I've passed the ratio in as a styles prop in StackedBarChart.types.ts file.

---
In reply to: [326304096](https://github.com/OfficeDev/office-ui-fabric-react/pull/10455#discussion_r326304096) [](ancestors = 326304096)